docs/topics/mta-assessment-changes-700.adoc

```diff
@@ -3,39 +3,39 @@
 // * docs/web-console-guide/master.adoc
 :_content-type: REFERENCE
 [id="mta-assessment-changes-700_{context}"]
-= Assessment module introduction
+= The Assessment module features
 
-The *Assessment* module in {ProductFullName} offers the following functionality for assessing and analyzing applications:
+The {ProductFullName} *Assessment* module offers the following features for assessing and analyzing applications:
 
-.Assessment hub
+Assessment hub::
+The *Assessment* hub integrates with the *Application inventory*.
 
-The *Assessment* hub integrates tightly with the *Application inventory*.
+Enhanced assessment questionnaire capabilities::
+In {ProductShortName} 7.0, you can import and export assessment questionnaires. You can also design custom questionnaires with a downloadable template by using the YAML syntax, which includes the following features:
 
-.Enhanced questionnaire capabilities
+* Conditional questions: You can include or exclude questions based on the application or archetype if a certain tag is present on this application or archetype.
+* Application auto-tagging based on answers: You can define tags to be applied to applications if a certain answer was provided.
```

Comment:
You can define tags to be applied to applications or archetypes if a certain answer was provided.